test/units/module_utils/facts/tests_date_time.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+import datetime
+
+from ansible.module_utils.facts.system.date_time import DateTimeFactCollector
+
+TIMESTAMP = datetime.datetime(2020, 7, 11, 12, 34, 56, 124356)
+
+
+@pytest.fixture
+def fake_now(monkeypatch):
+
+    class FakeNow:
+        @classmethod
+        def now(cls):
+            return TIMESTAMP
+
+    monkeypatch.setattr(datetime, 'datetime', FakeNow)
+
+
+@pytest.fixture
+def date_collector(monkeypatch, fake_now):
+    monkeypatch.setenv('TZ', 'Australia/Melbourne')
+
+    collector = DateTimeFactCollector()
+    data = collector.collect()
+
+    return data
```

Comment:
If you do this, you'll reduce the amount of copy-paste:
```suggestion
    return data['date_time']
```

Reply:
Thank you. It's not necessary now, because we use parameterize.

Reply:
I'd argue that it'd look cleaner and would better correspond to the fixture name that implies that it returns only the date-related subset of facts.